cmd/gapit/common.go

```diff
@@ -402,6 +404,130 @@ func getType(ctx context.Context, client service.Service, t *path.Type) (*types.
 	}
 }
 
+func printPodValue(ctx context.Context, v *pod.Value) error {
```

Comment:
Seems valuable in the `pod` package. While protobuf steals the `String` method from you, you can implement [`Format`](https://golang.org/pkg/fmt/#Formatter). There's plenty of examples knocking around in the codebase.